src/main/java/com/yahoo/identity/services/storage/sql/SqlSessionCreate.java

```diff
@@ -0,0 +1,50 @@
+package com.yahoo.identity.services.storage.sql;
+
+import com.yahoo.identity.services.session.SessionCreate;
+import org.joda.time.Instant;
+
+import javax.annotation.Nonnull;
+
+public class SqlSessionCreate implements SessionCreate {
```

Comment:
Since session shouldn't stored in db, we shouldn't implement `SessionCreate` in this way.

Reply:
Sorry for the confusion caused by class name(there is actually no interaction with DB for session and credential exception PasswordAbuseDection). Now all the session and credential implementations have been moved to its own folder.